Makefile

```diff
@@ -63,11 +85,23 @@ help:  ## Display this help
 ## Testing
 ## --------------------------------------
 
-.PHONY: test test-e2e
+ARTIFACTS ?= ${ROOT_DIR}/_artifacts
+
+.PHONY: test
 test: ## Run tests
 	go test -v ./...
-test-e2e: ## Launch integration e2e test with building images (for local usage)
-	$(MAKE) -C hack/tests test-e2e
+#test-e2e: ## Launch integration e2e test with building images (for local usage)
+#	$(MAKE) -C hack/tests test-e2e
```

Comment:
For the time being, I'm only disabling the old way of doing E2E tests because there is still some work to be done, but ideally, we should get to drop the old one as soon as we have tests using an external compute cluster.

Reply:
that makes sense to me